selftests/unit/test_utils_params.py

```diff
@@ -86,6 +86,96 @@ def testGetItemMissing(self):
     def testGetItem(self):
         self.assertEqual(self.params['image_size'], "10G")
 
+    def testCfgEnabled(self):
+        self.params["something"] = "no"
+        self.params["foobar"] = "yes"
+
+        self.assertEqual(False, params.is_cfg_enabled('something'))
+        self.assertEqual(False, params.is_cfg_enabled('notgiven'))
+        self.assertEqual(True, params.is_cfg_enabled('foobar'))
+
+    def testMultiParamNone(self):
+        self.assertEqual(0, len(self.params.get_multi_param("something")))
+
+    def testMultiParamOnlyOne(self):
+        self.params['something'] = 'foobar'
+
+        found = self.params.get_multi_param("something")
+        self.assertEqual(1, len(found))
+        self.assertEqual('foobar', found[0])
+
+    def testMultiParamMany(self):
+        self.params['something'] = 'foobar'
+        self.params['something2'] = 'foobar2'
+        self.params['something3'] = 'foobar3'
+
+        found = self.params.get_multi_param("something")
+        self.assertEqual(3, len(found))
+        self.assertEqual('foobar', found[0])
+        self.assertEqual('foobar2', found[1])
+        self.assertEqual('foobar3', found[2])
```

Comment:
note you can compare `self.assertEqual(["foobar", "foobar2", "foobar3"], found)`

Reply:
Good point, still less compatible with one-line versions that are easier to copy and paste like this should the number of unit tests grow. Also outdated though, please check the most recent unit tests from a few days ago.